runtime/gc_check/CheckEngine.cpp

```diff
@@ -52,6 +52,8 @@
 #include "ObjectModel.hpp"
 #include "ObjectAccessBarrier.hpp"
 #include "ScanFormatter.hpp"
+#include "SparseAddressOrderedFixedSizeDataPool.hpp"
+#include "SparseVirtualMemory.hpp"
```

Comment:
probably shoudl be around 

#if defined(J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION)